src/stability_sdk/animation_ui.py

```diff
@@ -271,43 +281,82 @@ def postprocess_video(fps: int, reverse: bool, interp_mode: str, interp_factor:
+
+            if upscale:
+                suffix += "_x2"
+                upscale_dir = os.path.join(outdir, "upscale") 
+                os.makedirs(upscale_dir, exist_ok=True)
+                frame_paths = glob.glob(os.path.join(outdir, "frame_*.png"))
+                num_frames = len(frame_paths)
+                if not can_skip_upscale:
+                    remove_frames_from_path(upscale_dir)
+                    for frame_idx in tqdm(range(num_frames)):
+                        frame = Image.open(frame_paths[frame_idx])
+                        frame = context.upscale(frame)
+                        frame.save(os.path.join(upscale_dir, os.path.basename(frame_paths[frame_idx])))
+                        yield {
+                            header: gr.update(value=format_header_html()) if frame_idx % 12 == 0 else gr.update(),
+                            image_out: gr.update(value=frame, label=f"upscale {frame_idx}/{num_frames}", visible=True),
+                            video_out: gr.update(visible=False),
+                            process_button: gr.update(visible=False),
+                            stop_button: gr.update(visible=True),
+                            error_log: gr.update(visible=False),
+                        }
+                        if interrupt:
+                            break
+                    last_upscale = upscale
+                outdir = upscale_dir
+
+            if interp_mode != 'None':
+                suffix += f"_{interp_mode}{interp_factor}"
+                interp_dir = os.path.join(outdir, "interpolate")
+                interp_mode = interpolate_mode_from_string(interp_mode)
+                if not can_skip_interp:
+                    remove_frames_from_path(interp_dir)
+                    num_frames = interp_factor * len(glob.glob(os.path.join(outdir, "frame_*.png")))
+                    for frame_idx, frame in enumerate(tqdm(interpolate_frames(context, outdir, interp_dir, interp_mode, interp_factor), total=num_frames)):
+                        yield {
+                            header: gr.update(value=format_header_html()) if frame_idx % 12 == 0 else gr.update(),
+                            image_out: gr.update(value=frame, label=f"interpolate {frame_idx}/{num_frames}", visible=True),
+                            video_out: gr.update(visible=False),
+                            process_button: gr.update(visible=False),
+                            stop_button: gr.update(visible=True),
+                            error_log: gr.update(visible=False),
+                        }
+                        if interrupt:
+                            break
+                    last_interp_mode, last_interp_factor = interp_mode, interp_factor
+                outdir = interp_dir
+
+            output_video = last_project_settings_path.replace(".json", f"{suffix}.mp4")
```

Comment:
cool! yesterday i was struggling to find the postprocessed video in the project folder so I had to click through all mp4. glad it's easier now!